src/fontra/client/core/ui-sliders.js

```diff
@@ -24,20 +26,19 @@ export class Sliders {
         divider.className = "slider-divider";
         this.container.appendChild(divider);
       } else {
-        const label = document.createElement("label");
-        const slider = document.createElement("input");
-        label.className = "slider-label";
-        slider.type = "range";
+        const slider = new RangeSlider();
+        slider.classList.add("slider");
+        slider.name = sliderInfo.name;
+        slider.minValue = sliderInfo.minValue;
+        slider.maxValue = sliderInfo.maxValue;
+        slider.defaultValue = sliderInfo.defaultValue;
         slider.step = "any";
-        slider.class = "slider";
-        slider.min = sliderInfo.minValue;
-        slider.max = sliderInfo.maxValue;
-        slider.value = sliderInfo.defaultValue;
-        slider.dataset.name = sliderInfo.name;
-        slider.oninput = (event) => this._dispatchListSelectionChanged();
-        label.appendChild(slider);
-        label.append(sliderInfo.name);
-        this.container.appendChild(label);
+        //TODO: make this dynamic also
+        slider.tickMarksPositions = [0, 100, 200, 300, 500, 700, 900];
+        slider.onChangeCallback = (event) => this._dispatchListSelectionChanged(); // TODO: not sure if that works
```

Comment:
Ah, that method is wrongly named. We should probably rename it to `_dispatchSlidersChangedEvent`